extensions/sharing/pkg/revaconfig/config.go

```diff
@@ -50,8 +50,8 @@ func SharingConfigFromStruct(cfg *config.Config) map[string]interface{} {
 						},
 						"cs3": map[string]interface{}{
 							"provider_addr":       cfg.UserSharingDrivers.CS3.ProviderAddr,
-							"service_user_id":     cfg.UserSharingDrivers.CS3.ServiceUserID,
-							"service_user_idp":    cfg.UserSharingDrivers.CS3.ServiceUserIDP,
+							"system_user_id":      cfg.UserSharingDrivers.CS3.SystemUserID,
+							"system_user_idp":     cfg.UserSharingDrivers.CS3.SystemUserIDP,
```

Comment:
the map keys must stay `service_user_*` as this is what the `cs3` driver expects